javascript/javascript2/week2/main.js

```diff
@@ -0,0 +1,92 @@
+console.log("Script loaded");
+
+const products = getAvailableProducts();
+
+const ul = document.querySelector('ul')
+
+const searchName = document.getElementById('itemSearch')
+const searchMinPrice = document.getElementById('minPrice')
+const searchMaxPrice = document.getElementById('maxPrice')
+
+// For Create Some Extra Feature
+
+const spiritAnimals = ['The Laughing Butterfly', 'The Brave Badger', 'The Sharp Hawk', 'The Sleeper Rabbit', 'The Bully Tiger', 'The Blind Eagle', 'The Big Eye Wolf', 'The Runner Turtle', 'The Walking Grasshopper', 'The Lazy Ant']
+
+products.forEach(product => {
+    const random = Math.floor(Math.random() * 10)
+    product.for = spiritAnimals[random]
+})
+
+function deleteMain() {
+    document.querySelector('ul').innerHTML = ''
+}
+
+
+function renderProducts(products) {
+    products.forEach(product => {
+        const li = document.createElement('li')
+        const itemName = document.createElement('h2')
+        const itemPrice = document.createElement('div')
+        const itemRatio = document.createElement('div')
+        const itemFor = document.createElement('div')
+
+        itemName.innerHTML = product.name
+        li.appendChild(itemName)
+
+        itemPrice.innerHTML = `<b>Price</b>: ${product.price}`
+        li.appendChild(itemPrice)
+
+        itemRatio.innerHTML = `<b>Rating</b>: ${product.rating}`
+        li.appendChild(itemRatio)
+
+        itemFor.innerHTML = `<b>For</b>: ${product.for}`
+        li.appendChild(itemFor)
+
+        ul.appendChild(li)
+    })
+}
```

Comment:
Great job.
Also try to avoid having div inside li's. 
I would use li for them since the itemName, itemPrice etc are all lists and append them to a ul.